catalog/security/banner-marking/src/test/groovy/org/codice/alliance/security/banner/marking/BannerCommonMarkingExtractorSpec.groovy

```diff
@@ -319,6 +312,114 @@ class BannerCommonMarkingExtractorSpec extends Specification {
         '//COSMIC RESTRICTED//BOHEMIA' | true            | null
     }
 
+    def 'test security conflict classification'() {
+        when:
+        initBannerMarkings("TOP SECRET")
+        metacard.setAttribute(new AttributeImpl(Security.CLASSIFICATION, "CONFLICTING VALUE"))
+        validationException = null
+        try {
+            extractor.processClassMarking(metacard, bannerMarkings)
+        } catch (RuntimeException rte) {
```

Comment:
❗️Instead of a `try/catch` in your `when` block, you should just call your method in the `when` block and then assert `thrown RuntimeException` in the catch block.

Also, asserting against the message is fragile.

This applies to all the tests where you've used this paradigm.